src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
+@import "./mixins.scss";
+@import "./functions.scss";
+
+$cell-size: 100px;
+$border-color: #000;
+$background-color: #eee;
+$font-family: arial;
+$font-size: 30px;
+$padding: 10px;
+$gap: 1px;
+$body-width: 800px;
+$days: mon, tue, wed, thu, fri, sat, sun;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  margin: 0 auto;
+  flex-wrap: wrap;
+  width: calc(7 * (#{$cell-size} + 2 * #{$gap} + 2 * #{$padding}));
```

Comment:
I think your calculations need some adjustments, let's simplify it:

7 cells + 6 gaps

No need for paddings